projects/admin/src/app/circulation/main-request/requested-item/requested-item.component.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * RERO ILS UI
+ * Copyright (C) 2019 RERO
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, version 3 of the License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { RecordService } from '@rero/ng-core';
+import { Item, LoanState } from '../../../class/items';
+
+@Component({
+  selector: 'admin-requested-item',
+  templateUrl: './requested-item.component.html',
+  styleUrls: ['./requested-item.component.scss']
+})
+export class RequestedItemComponent implements OnInit {
+
+  /** requested item */
+  @Input() item: Item;
+  /** Is the detail should be collapsed */
```

Comment:
You can add empty lines around here.

Reply:
I choose to group all Input/output into one "block". I think it's more readable (my feeling)